### ISSUE TITLE
PUTTYgen

### ISSUE THREAD

**Issue by mcgratta** [firemodels/fds#2680]
In the wiki GitHub User Workflow, there is a line

On a Windows-based computer you can use the PuTTYgen program from the [Putty package](http://www.chiark.greenend.org.uk/~sgtatham/putty/download.html) to get an SSH key.

Has anyone tested this. Apparently, PUTTYgen generates public ssh keys that are not accepted by GitHub.

**Comment by gforney** [comment]
we've got a wiki just for setting up two factor authentication at

https://github.com/firemodels/fds-smv/wiki/Two-Factor-Authentication

The lines you refer to in the work flow wiki should refer to it.  I'll make
the change.

I had forgotten but I couldn't get putty's ssh key's to work either.  I had
to use a personal access token which the 2 factor wiki talks about.

On Thu, Sep 3, 2015 at 12:44 PM, Kevin McGrattan notifications@github.com
wrote:

> In the wiki GitHub User Workflow, there is a line
> 
> On a Windows-based computer you can use the PuTTYgen program from the Putty
> package
> http://www.chiark.greenend.org.uk/%7Esgtatham/putty/download.html to
> get an SSH key.
> 
> Has anyone tested this. Apparently, PUTTYgen generates public ssh keys
> that are not accepted by GitHub.
> 
> —
> Reply to this email directly or view it on GitHub
> https://github.com/firemodels/fds-smv/issues/2680.

## 

Glenn Forney

**Comment by gforney** [comment]
I created a test git repo that contains just one file.  It will be a lot easier to use this to test ssh or other methods for connecting to github.  I added notes on using Putty ssh keys to the  [Two Factor Authentication wiki](https://github.com/firemodels/fds-smv/wiki/Two-Factor-Authentication) . I was able to get putty ssh keys to work with the test wiki.  If you are still having trouble let me know .  Maybe I left something out of the wiki.
